geonode/base/forms.py

```diff
@@ -484,7 +484,9 @@ def __init__(self, *args, **kwargs):
         self.user = kwargs.pop('user', None)
         super().__init__(*args, **kwargs)
         self.fields['regions'].choices = get_tree_data()
-
+        self.can_change_perms = self.user and (self.user.has_perm(
```

Comment:
@marthamareal doesn't self.user.has_perm already take into account if the user is an admin?